ltsa/lto_dump.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/ksh
+
+# Usage: KEYCLOAK_CLIENT_SECRET=your-secret ./lto_dump.sh [type]
+# type: 1 or 2 (required)
+
+# Configuration
+API_URL="${API_URL:-http://localhost:4007}"
```

Comment:
I would recommend to remove default value and make all this as mandatory env var, if not present error out